helm/app-admission-controller/values.yaml

```diff
@@ -4,6 +4,8 @@ image:
 
 Installation:
   V1:
+    Provider:
+      Kind: aws
```

Comment:
We have the values in ci/default-values.yaml so can we remove from here?

Otherwise if the config goes wrong we might validate the wrong provider.

Reply:
You are right, I was adding this because of chart validation from architect, but maybe values in CI are just enough.